app/components/chat/enable-gifs-button.hbs

```diff
@@ -1,7 +1,18 @@
-<button
+<div 
+  class="grid grid-cols-2"
   title={{if @enabled (t "chat.titles.img_on") (t "chat.titles.img_off") }}
-  class="cool-button"
-  type="button"
-  {{on "click" @toggleGifs}}>
-  {{if @enabled this.imgOn this.imgOff}}
-</button>
\ No newline at end of file
+>
+  <label
+    id="gif-enabler-label"
+    for="gif-enabler"
+  >
+    Show GIFs
```

Comment:
Guess it's technically show images? We might have a translation key for this already

Reply:
Went thru the translation file and there's only labels for the title that changes ("Showing Images" and "Not Showing Images"). But yeah the text shouldn't be hardcoded in there anyways